.github/workflows/test_dbt_models.yaml

```diff
@@ -3,7 +3,19 @@ name: test-dbt-models
 # This workflow is not scheduled or run on PRs because it is manually triggered
 # by the Data Department's sqoop data extraction process upon completion. See
 # https://github.com/ccao-data/service-sqoop-iasworld
-on: workflow_dispatch
+on:
+  workflow_dispatch:
+    inputs:
+      select:
+        description: dbt select statement for tests to run
+        required: false
+        default: --selector select_data_test_iasworld
+        type: string
+      upload_test_results:
+        description: Upload test results to S3
+        required: false
+        default: true
```

Comment:
Setting this to `true` by default means that we don't have to update `service-spark-iasworld` to handle this change.

Reply:
> Otherwise, won't we constantly be uploading new results every time we trigger this workflow?

Not necessarily, unless workflow callers are leaving it checked. But I hear your point that it's a better nudge to leave it unchecked, and it's an easy change to update `service-spark-iasworld` to provide the argument anyway. I made that change and will put up a corresponding `service-spark-iasworld` PR.